src/plugins/azgenerator/TemplateAzureCliReport.ts

```diff
@@ -67,6 +67,22 @@ function getCommandBody(model: CodeModelAz, needUpdate: boolean = false) {
     
     mo.push("");
 
+    // Command group
+    mo.push("#### Command group");
+    mo.push("|Name (az)|Name (swagger)|");
+    mo.push("|------|----|");
+    mo.push("|" + model.CommandGroup_Name + "|" + model.CommandGroup_CliKey + "|");
+    mo.push("");
+
+    // Command
+    mo.push("#### Command");
+    mo.push("|Name (az)|Name (swagger)|");
+    mo.push("|------|----|");
+    mo.push("|" + model.Command_MethodName + "|" + model.Command_CliKey + "|");
+    mo.push("");
```

Comment:
I think this part should moved into if(model.SelectFirstMethod) do while loop. In az one command can map to multiple method, the method is the operation in swagger. You may also add a column to indicate the method name

Reply:
Fixed and updated example. Thanks

Reply:
can we have this in command help directly instead of report?

Reply:
I think for operationGroup and operation we can, but for parameters we can't, help.py doesn't have all the parameters info just those actions and jsons, it's a little troublesome if we put it in two places

Reply:
but it's easier for CLI team to use, it shouldn't be too troublesome, is it?

Reply:
Maybe we can add both

Reply:
Add a new commit to include swagger in help if "--debug" is used. We show this to cli team so that they know how to use this. They can still use report.md if they do like use this flag. But I think as a developer, it's not a bug issue. ; )
Here's an example:
```
(clienv) D:\>az datafactory trigger create -h
This command is from the following extension: datafactory

Command
    az datafactory trigger create : Creates or updates a trigger. Command group swagger
    name=Triggers, Command swagger name=CreateOrUpdate.
        Command group 'datafactory trigger' is experimental and not covered by customer
        support. Please use with discretion.
Arguments
    --factory-name      [Required] : The factory name. Swagger name=factoryName.
    --name -n           [Required] : The trigger name. Swagger name=triggerName.
    --properties        [Required] : Properties of the trigger. Swagger name=properties Expected
                                     value: json-string/@json-file.
    --resource-group -g [Required] : Swagger name=resourceGroupName.
    --if-match                     : ETag of the trigger entity.  Should only be specified for
                                     update, for which it should match existing entity or can be *
                                     for unconditional update. Swagger name=If-Match.
```

Reply:
Isn't a little weird to only show swagger name in the --resource-group help info? I think the help info you add has overwrite the CLI inline type help info.

Reply:
You are right. I did not notice resource group, tags and location help info are generated within azure.cli. If we decide it in az, cli won't override it. For these 3 parameters I won't add swagger name, if user wants to know it, they can check report.md